### DISCUSSION TITLE
Use case with 100m resolution Zensus 2011 data (Germany)

### DISCUSSION THREAD

**Discussion by jgaffuri** [eurostat/gridviz#105] [Show and tell]
Hi, especially @wahlatlas !

Here is a new test case based on 100m resolution statistical grid on Germany: https://eurostat.github.io/gridviz/examples/DE.html

For such detailled resolution, the WMS map background at building and street scale with adaptative transparency really help showing the local geography under the statistical figures.

More to come... with statistics on buildings and habitation units from https://www.zensus2011.de/DE/Home/Aktuelles/DemografischeGrunddaten.html?nn=559100#Gitter

Note that the underlying prepared data and the transormation processes are shared in this repository: https://github.com/jgaffuri/tiled-grid-germany-zensus2011 It was preprocessed using [gridtiler](https://github.com/eurostat/gridtiler).

**Comment by jgaffuri** [comment]
I revamped the map: https://eurostat.github.io/gridviz/examples/DE.html
It now shows many of the indicators available at 100m resolution.
The data preparation/tiling process is also now fully in python, based on [panda](https://pandas.pydata.org/) and [gridtiler](https://github.com/eurostat/gridtiler/): https://github.com/jgaffuri/tiled-grid-germany-zensus2011

@wahlatlas Feel free to reuse it to promote this amazing gridded dataset !

**Comment by jgaffuri** [comment]
I added a style for smoothing total population. See there: https://eurostat.github.io/gridviz/examples/DE.html?lay=popS

![aaa](https://github.com/eurostat/gridviz/assets/4315279/b22a72e3-afaa-4159-9579-47cbbbe80d08)

**Reply by wahlatlas** [reply]
Smoothing is interesting as the reality does not adhere to gridcell boundaries. However I feel in this example I can adjust the slider to levels that exaggerate population values overall
